### DISCUSSION TITLE
The SNI hostname is required when hostname validation is enabled

### DISCUSSION THREAD

**Discussion by toas-jyskebank-dk** [quarkusio/quarkus#39475] [Community]
I am trying to upgrade my Quarkus application from version 3.3.2 to 3.8.2

When I try to start the application I get the following error, preventing me from starting the application

`org.infinispan.commons.CacheConfigurationException: ISPN004112: The SNI hostname is required when hostname validation is enabled`

I'm aware, that the setting `quarkus.infinispan-client.ssl-host-name-validation` has changed its default value to true, so I set the value to false in my application.properties.

So, it seems, that this specific setting is being ignored on startup.

Anyone else experiencing that?

**Comment by sberyozkin** [comment]
@toas-jyskebank-dk Can you please create an issue from this discussion and add an infinispan label ?

**Comment by toas-jyskebank-dk** [comment]
This turned out to be an error on my side. I'm closing this conversation.
